### ISSUE TITLE
Training library (updated version)

### ISSUE THREAD

**Issue by eeko-kon** [iomega/ms2query#196]
Hi Niek! I updated my scripts to fit the new ones for library training: 

```
from ms2query.create_new_library.library_files_creator import LibraryFilesCreator
from ms2query.clean_and_filter_spectra import clean_normalize_and_split_annotated_spectra
from ms2query.utils import load_matchms_spectrum_objects_from_file, select_files_in_directory
from ms2query.run_ms2query import download_zenodo_files
from ms2query.ms2library import select_files_for_ms2query


spectrum_file_location =  # The file location of your library spectra
ionisation_mode = #The ionisation mode, choose between "positive" or "negative"
directory_for_library_and_models = # Fill in the direcory in which the models will be downloaded and the library will be stored.

# Downloads the models:
download_zenodo_files(ionisation_mode, directory_for_library_and_models, only_models=True)
library_spectra = load_matchms_spectrum_objects_from_file(spectrum_file_location)

files_in_directory = select_files_in_directory(directory_for_library_and_models)
dict_with_file_names = select_files_for_ms2query(files_in_directory, ["s2v_model", "ms2ds_model", "ms2query_model"])
ms2ds_model_file_name = dict_with_file_names["ms2ds_model"]
s2v_model_file_name = dict_with_file_names["s2v_model"]
ms2query_model = dict_with_file_names["ms2query_model"]

# Fill in the missing values:
cleaned_library_spectra = clean_normalize_and_split_annotated_spectra(library_spectra, ion_mode_to_keep="")[0]

library_creator = LibraryFilesCreator(cleaned_library_spectra,
                                      output_directory=directory_for_library_and_models, 
                                      ms2ds_model_file_name=ms2ds_model_file_name, 
                                      s2v_model_file_name=s2v_model_file_name, ) 
library_creator.create_all_library_files()
```

While importing the libraries, I am getting this error:
```
from matchmsextras.pubchem_lookup import pubchem_metadata_lookup
ImportError: cannot import name 'pubchem_metadata_lookup' from 'matchmsextras.pubchem_lookup'
```

**Comment by eeko-kon** [comment]
Even running ms2query from the CLI directly produces the same error:

`ms2query -h 
`

```
from matchmsextras.pubchem_lookup import pubchem_metadata_lookup
ImportError: cannot import name 'pubchem_metadata_lookup' from 'matchmsextras.pubchem_lookup' (/Users/eeko/Desktop/snakemake_UmetaFlow/.snakemake/conda/119947466b15fa285b27c7a511449267_/lib/python3.8/site-packages/matchmsextras/pubchem_lookup.py)
```

**Comment by justinjjvanderhooft** [comment]
I think matchmsextras was updated - Niek is on holidays but upon his return he could look into it. @florian-huber - have you recently worked on matchmsextras?

**Comment by justinjjvanderhooft** [comment]
Ah, thanks, very helpful @eeko-kon and glad you could get it back to run!

**Comment by eeko-kon** [comment]
At the moment, I can make it run by omitting the specific function, by setting the do_pubchem_lookup to False:

    `cleaned_library_spectra = clean_normalize_and_split_annotated_spectra(library_spectra, do_pubchem_lookup=False, ion_mode_to_keep=ion_mode)[0] `

but I did not fix the file, since there are a lot of changes. I think Niek would be much quicker than me :) 

After his holidays of course!

**Comment by niekdejonge** [comment]
Hi Efi, thanks for making the clear issue. Sorry for the long wait, I did just return from my holidays. This shows that I should not make any changes to our packages just before my holidays ;) But glad to hear that you were able to circumvent the issue somewhat. 
I was under the impression that the matchmsextras version was fixed in the setup.py of ms2query, so that these changes would not affect MS2Query. But this was wrong, so the new version was used, which was not compatible. For now, I will make a quick new release that just fixes the version of matchmsextras. I will later implement the new version correctly in ms2query (since this is a lot faster and more robust). 
Thanks again for pointing this out and sorry for the inconvenience.
